src/lib/serviceRoutines/statisticsTreat.cpp

```diff
@@ -368,10 +368,18 @@ std::string statisticsTreat
     out += TAG_ADD_COUNTER("logRequests", noOfLogRequests);
   }
 
+#if 0
+  //
+  // The valgrind test suite uses REST GET /version to check that the broker is alive
+  // This fact makes the statistics change and some working functests fail under valgrindTestSuite
+  // due to the 'extra' version-request in the statistics.
+  // Removing version-requests from the statistics 'report' fixes this problem.
+  //
   if (noOfVersionRequests != -1)
   {
     out += TAG_ADD_COUNTER("versionRequests", noOfVersionRequests);
   }
+#endif
```

Comment:
No, both are needed.
Without REGEX((1|2)) it can never work, due to huge changes in timing with/without valgrind.

Removing the versionRequests from the statistics output is something else (that's why I say 'Apart from ...').
The problem is stat the statistics-output of ZERO version-requests get no line at all about version-requests, but if there has been one or more version-requests, then we have a line about it in the statistics-output.  I don't know how to express "Line or no line" in REGEX.
That is basically the reason for this '#if 0/#endif'.